model-engine/model_engine_server/api/v2/batch_completion.py

```diff
@@ -149,5 +156,5 @@ async def cancel_batch_completion(
         )
         raise HTTPException(
             status_code=500,
-            detail="Internal server error",
+            detail=f"Internal server error. request_id: {request_id}",
```

Comment:
tbh, we should update all of these `Internal server error` to be explicit about what exact error it is. there's no need to hide it from users

Reply:
yes i agree.. i don't want to address this right now though